app/javascript/mastodon/components/status_action_bar.js

```diff
@@ -116,17 +116,23 @@ export default class StatusActionBar extends ImmutablePureComponent {
 
   render () {
     const { status, me, intl, withDismiss } = this.props;
-    const reblogDisabled = status.get('visibility') === 'private' || status.get('visibility') === 'direct';
+
+    const reblogDisabled     = status.get('visibility') === 'private' || status.get('visibility') === 'direct';
```

Comment:
- `['private', 'direct'].includes(status.get('visibility'))`
- or use `publicStatus` for reblog button too?

Reply:
I agree. b6bc536